cmd/influxd/server_integration_test.go

```diff
@@ -280,12 +286,13 @@ func runTestsData(t *testing.T, testName string, nodes Cluster, database, retent
 	// The tests. Within these tests %DB% and %RP% will be replaced with the database and retention passed into
 	// this function.
 	tests := []struct {
-		reset    bool   // Delete and recreate the database.
-		name     string // Test name, for easy-to-read test log output.
-		write    string // If equal to the empty string, no data is written.
-		query    string // If equal to the blank string, no query is executed.
-		queryDb  string // If set, is used as the "db" query param.
-		expected string // If 'query' is equal to the blank string, this is ignored.
+		reset         bool   // Delete and recreate the database.
+		name          string // Test name, for easy-to-read test log output.
+		write         string // If equal to the empty string, no data is written.
+		query         string // If equal to the blank string, no query is executed.
+		queryDb       string // If set, is used as the "db" query param.
+		expected      string // If 'query' is equal to the blank string, this is ignored.
+		expectPattern string // Regexp alternative to expected field.
```

Comment:
Minor: I would called it `expectedPattern` so it lined up with `expected`, or perhaps renamed `expected` to `expect`.